server/db/schema.js

```diff
@@ -110,6 +112,34 @@ const schema = {
       )`,
     },
   ],
+  phaseTables: [
+    {
+      definition: `CREATE TABLE IF NOT EXISTS ${collections.GLOBAL_PHASE} (
+        id serial primary key,
+        name varchar(255) not null,
+        start_date date not null,
+        end_date date not null,
+        created_by varchar(255) not null,
+        updated_by varchar(255) not null,
```

Comment:
updated fields should be nullable

Reply:
I can do that, but FYI when they create the phase, it uses the create user/time for the updated fields, so it never is null. I figured that creator makes sense as updater, because if I wanted to query by most recently updated that I would want to use the created date if it hadn't been edited yet.